cmd/access/node_builder/access_node_builder.go

```diff
@@ -988,6 +990,9 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 			if builder.stateStreamConf.ResponseLimit < 0 {
 				return errors.New("state-stream-response-limit must be greater than or equal to 0")
 			}
+			if builder.stateStreamConf.MaxRegisterIDsPerMsg == 0 {
+				return errors.New("state-stream-max-register-values must be greater than or equal to 0")
```

Comment:
this check returns an error if the setting is equal to 0, but the error message says `must be greater than or equal to 0`. which is correct?

Reply:
changed to just `greater than 0`